src/generate_price_reconciliation_report.py

```diff
@@ -0,0 +1,264 @@
+from configs.constants import TableNames
+import os
+import polars as pl
+import sqlite3
+from utils.utils import Utils
+
+
+def get_config_n_constant() -> tuple[str, str, str, str, str, str, str]:
```

Comment:
An unusual return signature: shows that this would be simpler as a dict or something, you don't really do anything useful by gathering the constants together.